### PR TITLE
Feature todo

### PR DESCRIPTION
**Pull request by andresporras3423**
- The website was built with JavaScript and Webpack
- The page let the user create to-dos for different projects
- Index page contains 2 tabs, the first one let you add new to-dos to any project
- First tabs also show a list of to-dos for the selected project
- User can watch, edit or delete a project
- Second tab is to create new projects
- Second tab has a similar structure to the first one, a form to add projects and a table with all the projects.
- You can also watch, edit, and delete projects.
- The project has the next folders: load, logic, data, and classes 
- The load folder contains the modules to edit the DOM
- The logic folder contains the functions to work with the data
- The data folder contains the projects, to-dos, and the icons
- The classes folder contains the todo class and project class

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+    "eslint-config-airbnb-base": "^14.1.0",
+    "eslint-plugin-import": "^2.20.2",
+    "sass": "^1.29.0",
+    "sass-loader": "^10.1.0",
+    "stylelint": "^13.3.3",
+    "stylelint-config-standard": "^20.0.0",
+    "stylelint-csstree-validator": "^1.9.0",
+    "stylelint-scss": "^3.17.2",
+    "webpack": "^5.9.0",
+    "webpack-cli": "^4.2.0"
```
Comment (addod19) [inline package.json]: Webpack dev server is missing from the devDependencies, hence outputs `webpack-dev-server` not found when the command `npm run start` is executed. Let us fix this :smile:

**Review by addod19 — CHANGES_REQUESTED** [review]
## Status: Require changes :recycle: :traffic_light: 

Hi @andresporras3423, great job on this project :+1: The PR is very descriptive :fireworks: README file professional :1st_place_medal: And the project works as expected :+1: Awesome. However, before we proceed, let us kindly address the comments below:

- Let us implement localStorage to save projects and todos between sessions. Currently, when the browser refreshes all the projects and todos are deleted. Let us fix that
- The `npm run` command outputs an error with the server as outlined below in the comments below

Once these are fixed, kindly submit another code review request :smile: 
_Happy coding_ :man_technologist: 
[Daniel Larbi Addo](linkedin.com/in/daniel-larbi-addo/)

**Comment by andresporras3423** [comment]
Thank you TSE for the comments, I have made the corrections,  now:
1) Project is using localStorage
2) "npm run start" command is working

**Review by Torres-ssf — APPROVED** [review]
### PR Approved :+1: 
Hi @andresporras3423  :hand: 
Awesome job on this project.
You may merge the PR.
Good luck with the next project.

Happy coding :computer: 
**Sergio Torres**

**Pull request merged**